quicktest.py

```diff
@@ -69,6 +69,22 @@ def run_tests(self):
 
         django.setup()
 
+        # Workaround for incompatibility between SQLite 3.36+ and SpatiaLite 5,
```

Comment:
```suggestion
        # Workaround for incompatibility between SQLite 3.36+ and SpatiaLite 5.0,
```

I've not tested many version combinations but it seems that SpatiaLite 5.0 is the issue. The tests pass for me with SpatiaLite 5.1 and SQLite 3.45.2 (appreciate it is also a much more modern version of SQLite)

Reply:
Thank you for testing. Yeah, I guess to update SpatiaLite, we would need to update the Ubuntu version on GitHub Actions, when 24.04 becomes available.

Are there any clear release notes for SpatiaLite 5.1? I found the site hard to navigate.

Reply:
I couldn't find any release notes either.